docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+
+:::note
+
+In accordance with GDPR and other legislations, these are only requests. **You** determine how those are handled and answered, manually or automatically.
+
+:::
+
+If, for example, your users want to make an `Access` request, and click on the `Access` button, they will be able to ask you to give them access to all the data you have on them, or to only select a subset of it.
+
+![First screen for an Access Request in the PRCI component](/img/captures/prci-access-01.png)
+![Second screen for an Access Request in the PRCI component](/img/captures/prci-access-02.png)
+
+Depending on your configuration, two scenarios can then happen.
+
+If like in this example, you set up everything to automatically treat the request, then your users will directly see their request granted (or rejected, depending on the parameters you defined).
+
+![Status of a Delete Request as shown in the PRCI component: under review](/img/captures/prci-access-request-status-granted.png)
+
+Other scenarios might however require to be manually handled.
+In this case, your users will see their request is "pending" (i.e. waiting to be handled by your organization).
+
+In this demo, you can see this happen when sending a `delete` request:
+
+![Status of a Delete Request as shown in the PRCI component: under review](/img/captures/prci-delete-request-status-under-review.png)
+
+<!-- TODO: add link to User doc -->
+
+## 4. Add an Admin Interface
+
+An additional Privacy Component for the Web, called the "Data Consumer Interface", is here to help you handle these requests.
+It offers a simple interface to allow relevant members of your organization (e.g. system administrators or DPOs) to manage all privacy requests related matters.
+
+To add the Data Consumer Interface Web Component to your project, simply follow the same steps as for the Privacy Request Capture Interface one, but with the `@blindnet/dci` npm package, and `<bldn-data-consum>` HTML custom element.
+
+```html
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta charset="utf-8" />
+    <title>Privacy Components for the Web - Minimal Static Demo</title>
+  </head>
+  <body>
+    <h1>Privacy Components for the Web - Minimal Static Demo</h1>
+
+    <h2>Assert Your Privacy Rights</h2>
+
+    <bldn-priv-request></bldn-priv-request>
+
+    <h2>Manage Your Users' Requests</h2>
+
+    <bldn-data-consum></bldn-data-consum>
+
+    <script src="https://unpkg.com/@blindnet/prci/dist/index.core.min.js?module" type="module" ></script>
+    <script src="https://unpkg.com/@blindnet/dci/dist/index.core.min.js?module" type="module" ></script>
+  </body>
+</html>
+```
+
+<!-- TODO: add link to API doc and more details -->
+
+## 5. Answer Your Users' Requests
+
+> :construction_worker: redaction in progress
+
+Open your web app / page, and follow the instructions given in the Data Consumer Interface Web Component.
+
+## 6. Set-up Storage
+
+> :construction_worker: redaction in progress
```

Comment:
```suggestion
> :construction_worker: writing in progress
```
redaction in EN means "the censoring or obscuring of part of a text for legal or security purposes."

Please make the same change elsewhere

Reply:
Addressed in #40